setup.py

```diff
@@ -54,5 +54,5 @@
         'Topic :: Software Development :: Quality Assurance',
         'Topic :: Software Development :: Testing'
     ],
-    install_requires=['selenium>=3.14.1']
+    install_requires=['selenium<4']
```

Comment:
how about changing like: `selenium>=3.14.1,<4`?

Reply:
Thanks, it's more accurate.

```
(--snip--)
Searching for selenium<4,>=3.14.1
Reading https://pypi.python.org/simple/selenium/
Downloading https://files.pythonhosted.org/packages/80/d6/4294f0b4bce4de0abf13e17190289f9d0613b0a44e5dd6a7f5ca98459853/selenium-3.141.0-py2.py3-none-any.whl#sha256=2d7131d7bc5a5b99a2d9b04aaf2612c411b03b8ca1b1ee8d3de5845a9be2cb3c
Best match: selenium 3.141.0
(--snip--)
```

https://stackoverflow.com/questions/11472810/how-to-specify-version-ranges-in-install-requires-setuptools-distribute